robotseo/robot.go

```diff
@@ -21,12 +22,21 @@ func Init(pathToRobotFile string) {
 	var b []byte
 	var err error
 	var fileName string
-	if !strings.HasSuffix(pathToRobotFile, "/") {
+
+	if !strings.HasSuffix(pathToRobotFile, "/") && pathToRobotFile != "" {
 		pathToRobotFile += "/"
 	}
 	for _, lang := range []config.Language{config.English, config.Welsh} {
 		fileName = "robot_" + lang.String() + ".json"
-		b, err = os.ReadFile(pathToRobotFile + fileName)
+
+		// if pathToRobotFile is empty (the default) we get the robot file for the component tests
+		// otherwise we get the robot file from the local file store and the path specified by pathToRobotFile
+		if pathToRobotFile == "" {
```

Comment:
Is this logic to fix the component test or something else? The comment suggests it's to fix tests but this package sits outside the tests?

Reply:
Thank you Nick. This is now addressed.